waku/v2/node/jsonrpc/store_api.nim

```diff
@@ -27,7 +27,10 @@ proc installStoreApiHandlers*(node: WakuNode, rpcsrv: RpcServer) =
       debug "get_waku_v2_store_v1_messages response"
       responseFut.complete(response.toStoreResponse())
     
-    let historyQuery = HistoryQuery(topics: topics,
+    var contentFilters: seq[HistoryContentFilter] = @[]
+    for topic in topics:
```

Comment:
I'd probably either add some comment here or be more explicit with contentTopic here, otherwise we can easily introduce bugs in future with topic/contenttopic confusion

Reply:
Sure, addressed!